boards/arm/degu_evk/Kconfig.defconfig

```diff
@@ -0,0 +1,86 @@
+# Kconfig - Degu Evaluation Kit configuration
+#
+# Copyright (c) 2019 Atmark Techno, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_DEGU_EVK
+
+config BOARD
+	default "degu_evk"
+
+if USB
+
+config USB_NRF52840
+	default y
+
+config USB_DEVICE_STACK
+	default y
+
+endif # USB
+
+if IEEE802154
+
+config IEEE802154_NRF5
+	default y
+
+endif # IEEE802154
+
+if I2C_NRFX
```

Comment:
I'd recommend to use `if I2C` here, for consistency with other boards with nRF SoCs, and because this is actually what you want to do here - enable instances 0 and 1 when I2C is enabled, not necessarily when a particular type of I2C driver is enabled. Currently for nRF SoCs there is no other choice in the Zephyr source tree, only nrfx based drivers can be used, but in theory this may change in some future.

Same story below with `if ADC_NRFX_SAADC` - IMO `if ADC` would be a better choice.

Reply:
I missed it...thank you!